.size-limit.json

```
@@ -8,7 +8,7 @@
       "!dist/*.webmanifest",
       "!dist/*.png"
     ],
-    "limit": "60 KB"
+    "limit": "60.57 KB"
```

```suggestion
    "limit": "65 KB"
```

I am using 5 KB step for limit in this project